modular_citadel/code/_DEFINES/construction.dm

```diff
@@ -0,0 +1 @@
+#define MAT_PLASTIC		"$plastic"
```

Comment:
you defined it earlier and here, why

Reply:
Rogue file I forgot to delete. Thought modularizing it would work but it didn't.

Reply:
it wouldn't work in modular_citadel because defines only work /after/ they're loaded in so it has to be loaded before /code folder (minus /code/__DEFINES/*) loads.